graylog2-server/src/test/java/org/graylog/plugins/pipelineprocessor/functions/FunctionsSnippetsTest.java

```diff
@@ -1108,24 +1119,32 @@ void conversions() {
         assertThat(message.getField("double_inf2")).isEqualTo(Double.NEGATIVE_INFINITY);
         assertThat(message.getField("double_inf3")).isEqualTo(Double.POSITIVE_INFINITY);
         assertThat(message.getField("double_inf4")).isEqualTo(Double.NEGATIVE_INFINITY);
+        assertThat(message.getField("double_default_null_set_single_field")).isNull();
+        assertThat(message.getField("double_default_null_set_single_field")).isNull();
```

Comment:
are these supposed to be for different fields?

Reply:
Thanks for seeing that. Yep, that was supposed to be a different field. Just fixed it.